t/batches.t

```diff
@@ -170,9 +171,19 @@ subtest 'RPCAPI get_batch_job_result' => sub {
 
         is( $res->{nb_running}, @domains, 'correct number of runninng tests' );
         is( $res->{nb_finished}, 0, 'correct number of finished tests' );
+
+        my $res = $rpcapi->batch_status( { batch_id => $batch_id } );
+
+        is( $res->{waiting_count}, @domains, 'correct number of runninng tests' );
+        is( $res->{running_count}, 0, 'correct number of finished tests' );
+        is( $res->{finished_count}, 0, 'correct number of finished tests' );
+        is( $res->{waiting_tests}, undef, 'correct undefined list of waiting tests' );
+        is( $res->{running_tests}, undef, 'correct undefined list of running tests' );
+        is( $res->{finished_tests}, undef, 'correct undefined list of finished tests' );
```

Comment:
Maybe make the test explicit on whether these values are expected to be absent or present-but-undef?

There are more of these below.

Reply:
Updated.